AutomaticClockOuts/clock_out.js

```diff
@@ -25,31 +24,25 @@ const doClockOut = async (context) => {
 
   let totalClockOutsExecuted = 0;
   const usersWithClockOut = [];
+  const ClockOutMessage = `OMG, you have been working more than 12 hours in a row. \nPlease take a break and visit https://timetracker.ioet.com/ to set the right end time for your entries, we just did a clock out for you :wink:`;
```

Comment:
Maybe you could create a file called `constants.js` or something similar, to store constants like `ClockOutMessage` and `channel` to centralize these variables, what do you think?

Reply:
That's a good idea, I will make those changes.